public/app/features/manage-dashboards/DashboardImportPage.tsx

```diff
@@ -148,10 +149,12 @@ class UnthemedDashboardImport extends PureComponent<Props> {
                 label={
                   <Label className={styles.labelWithLink} htmlFor="url-input">
                     <span>
-                      Find and import dashboards for common applications at{' '}
-                      <TextLink variant="bodySmall" href="https://grafana.com/grafana/dashboards/" external>
-                        grafana.com/dashboards
-                      </TextLink>
+                      <Trans i18nKey="dashboard-import-page.field-label">
+                        Find and import dashboards for common applications at{' '}
+                        <TextLink variant="bodySmall" href="https://grafana.com/grafana/dashboards/" external>
+                          grafana.com/dashboards
+                        </TextLink>
+                      </Trans>
```

Comment:
this is a little tricky, because we don't actually want to translate the url here.

```suggestion
                      <Trans i18nKey="dashboard-import.gcom-field.label">
                        Find and import dashboards for common applications at <GcomDashboardsLink />
                      </Trans>
```

if you then create a new variable just inside `renderImportForm` like:
```typescript
const GcomDashboardsLink = () => (
  <TextLink variant="bodySmall" href="https://grafana.com/grafana/dashboards/" external>
    grafana.com/dashboards
  </TextLink>
);
```

that will then not translate the link. hope that makes sense 🙏